src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
(relink) I'm not sure why the cli-error and cli-help wouldn't be public interfaces, they would seem to be considering tk/main strips the namespaces off of the keys so that any namespace could use them. Also I would argue that it's a public interface because the `cli!` function uses that interface, exposed from a separate project, kitchensink.